packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkImplementation.kt

```diff
@@ -54,16 +54,21 @@ class DdSdkImplementation(
      */
     fun initialize(configuration: ReadableMap, promise: Promise) {
         val ddSdkConfiguration = configuration.asDdSdkConfiguration()
-        val credentials = buildCredentials(ddSdkConfiguration)
-        val nativeConfiguration = buildConfiguration(ddSdkConfiguration)
+        val sdkConfiguration = buildSDKConfiguration(ddSdkConfiguration)
+        val rumConfiguration = buildRUMConfiguration(ddSdkConfiguration)
```

Comment:
just for the naming consistency:

```suggestion
        val sdkConfiguration = buildSdkConfiguration(ddSdkConfiguration)
        val rumConfiguration = buildRumConfiguration(ddSdkConfiguration)
```